hudi-common/src/main/java/org/apache/hudi/metadata/MetadataPartitionType.java

```diff
@@ -434,6 +434,18 @@ public static List<MetadataPartitionType> getEnabledPartitions(TypedProperties w
         .collect(Collectors.toList());
   }
 
+  /**
+   * Returns the list of metadata partition types enabled and initialized based on the metadata config and table config.
+   */
+  public static List<MetadataPartitionType> getEnabledAndInitializedPartitions(TypedProperties writeConfig, HoodieTableMetaClient metaClient) {
```

Comment:
Maybe just rename to `getInitializedPartitions` because an initialized partition is definitely enabled.